vhpc_toolkit/config_objs.py

```diff
@@ -974,6 +974,40 @@ def create_svs(self, svs_name, vmnic, num_ports=8):
         host_network_obj = self.host_obj.configManager.networkSystem
         host_network_obj.AddVirtualSwitch(vswitchName=svs_name, spec=svs)
 
+    def modify_sriov(
+        self,
+        device_id: str,
+        num_virtual_functions: int = None,
+        enable_sriov: bool = True,
+    ):
+        """
+        Function to enable/disable SRIOV and/or change the number of virtual functions on devices of a host.
+        Number of virtual functions argument is skipped if user is trying to disable SRIOV
+        Args:
+            device_id: Address of the device which supports SRIOV in format xxxx:xx:xx.x
```

Comment:
device_id: PCIe address of the Virtual Function (VF) of the SRIOV device in format xxxx:xx:xx.x